src/shared/config/config.go

```diff
@@ -19,6 +19,12 @@ const (
 	PrometheusMetricsPortDefault = 2112
 	TelemetryErrorsAPIKeyKey     = "telemetry-errors-api-key"
 	TelemetryErrorsAPIKeyDefault = "d86195588a41fa03aa6711993bb1c765"
+	EnableTCPKey                 = "enable-tcp"
+	EnableTCPSnifferDefault      = true
+	EnableSocketScannerKey       = "enable-socket-scanner"
+	EnableSocketScannerDefault   = true
+	EnableDNSKey                 = "enable-dns"
+	EnableDNSSnifferDefault      = true
```

Comment:
So we're having flags for each type now ?

Reply:
Yes, it's can be helpful for debugging, shouldn't be used for anything else